src/source.css

```diff
@@ -3125,6 +3125,25 @@ pre:hover .copybutton {
     color: var(--soft-bright-color) !important;
 }
 
+/* Invite as guest in VC invite */
+.temporaryMembershipContainerBackground_cedfaf {
+	background-color: var(--dark-bg) !important;
+}
+
+.temporaryMembershipContainerBackground_cedfaf::before {
+    background-color: transparent !important;
+}
+
+/* Copy button in VC invite */
+.input_f8bc55.flexCenter_ec1a20 {
+    background-color: var(--hover-color) !important;
+}
+
+/* Hover on users color in vc invite */
+.inviteRow_cedfaf:hover {
+	background-color: var(--hover-color) !important;
+}
+
```

Comment:
I'd rename that section to something more easily searchable, for when you would no longer remember the exact wording, like "Voice channel invite".

A comment for every element seems a bit unnecessary, just grouping elements under the main one would suffice, while also making it easier to discern as its own section, when going over the file searching for things.